### DISCUSSION TITLE
Next release

### DISCUSSION THREAD

**Discussion by bbenameur** [elsa-workflows/elsa-core#4902] [General]
Hello,
Do you have any idea when the next release will be published , i need to use IWorkflowRegistry and its not existe on the 3.0.5?
Thanks

**Comment by glime-ai** [comment]
As of my last update, I do not have access to real-time data or the ability to provide current release dates for software, including Elsa Workflow. However, you can typically find information about upcoming releases and their timelines on the official Elsa Workflow GitHub repository under "Releases" or within the project's documentation.

To get the most accurate and up-to-date information regarding the release that includes `IWorkflowRegistry`, I recommend checking the Elsa Workflow GitHub repository for any announcements or updates on future releases. Additionally, you can engage with the community on their GitHub discussions or issues pages to ask the developers directly about the release schedule.

If `IWorkflowRegistry` is a new feature or enhancement, it might also be discussed in the issues or pull requests sections of the repository. Keep an eye on those sections as well for any news related to this interface.
